### PR TITLE
mohammad/new_trading

### PR DESCRIPTION
**Pull request by binary-hamedanchi**

### CODE REVIEW

**Comment by eugene-eeo** [comment]
very rough look over the code, maybe my final code review 😁 `not_first_request`, well... it doesn't feel right. maybe we should use `is_first_request`? that way you can just set it equal to true initially, (saving one line! of code) and it reads better.

**Comment by binary-hamedanchi** [comment]
@eugene-eeo  I agree. I meant to change it, but I've forgotten. Thanks for reminding me.
Wish you the best.

File: src/sass/pages/trading.scss [hunk 1]
```diff
@@ -8,27 +8,22 @@
...
     color: $COLOR_WHITE;
-    border: solid 1px $COLOR_ORANGE;
-    border-radius: 0;
-    border-bottom-left-radius: 4px;
-    background: $COLOR_ORANGE;
-    font-size: 24px;
-    font-weight: bold;
-    line-height: 0px;
+    line-height: 35px;
+    font-size: 26px;
```
Comment (negar-binary) [inline src/sass/pages/trading.scss]: is there a font-size from constants we can use?

Comment (binary-hamedanchi) [inline src/sass/pages/trading.scss]: Done. Thanks.

**Pull request merged**